### PR TITLE
Updated "construct JSON to return"

### PR DESCRIPTION
**Pull request by MisterKangaroo**
Changed maximum zerocoins per spend in the RPC_WALLET_ERROR to 20

### CODE REVIEW

**Comment by MisterKangaroo** [comment]
Pls check if it makes sense... since there is normally no limit on mints or spends

**Comment by presstab** [comment]
@MisterKangaroo it would be better practice to change this to check against `Params().Zerocoin_MaxSpendsPerTransaction()`

**Comment by MisterKangaroo** [comment]
Changed it... hope it makes sense... and wasnt absolutly misunderstood.
Pls double check.

File: src/wallet/rpczerocoin.cpp [hunk 1]
```diff
@@ -739,8 +739,8 @@ UniValue spendzerocoinmints(const JSONRPCRequest& request)
     UniValue arrMints = params[0].get_array();
     if (arrMints.size() == 0)
         throw JSONRPCError(RPC_WALLET_ERROR, "No zerocoin selected");
-    if (arrMints.size() > 7)
-        throw JSONRPCError(RPC_WALLET_ERROR, "Too many mints included. Maximum zerocoins per spend: 7");
+    if (arrMints.size() > Params().Zerocoin_MaxSpendsPerTransaction())
+        throw JSONRPCError(RPC_WALLET_ERROR, "Too many mints included. Maximum zerocoins per spend: Params().Zerocoin_MaxSpendsPerTransaction()");
```
Comment (presstab) [inline src/wallet/rpczerocoin.cpp]: You would print the string as:
`throw JSONRPCError(RPC_WALLET_ERROR, strprintf("Too many mints included. Maximum zerocoins per spend: %d", Params().Zerocoin_MaxSpendsPerTransaction());`

Comment (MisterKangaroo) [reply src/wallet/rpczerocoin.cpp]: Thanks Presstab!

**Pull request merged**